aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/transformations/DataTableTypeTransformations.java

```diff
@@ -1,19 +1,20 @@
 package aquality.selenium.template.cucumber.transformations;
 
 import aquality.selenium.template.models.ContactUsInfo;
-import io.cucumber.java.DataTableType;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.java.DefaultDataTableEntryTransformer;
 
+import java.lang.reflect.Type;
 import java.util.Map;
 
 public class DataTableTypeTransformations {
 
-    @DataTableType
-    public ContactUsInfo getContactUsInfo(Map<String, String> entry) {
-        ContactUsInfo contactUsInfoModel = new ContactUsInfo();
-        contactUsInfoModel.setName(entry.get("Name"));
-        contactUsInfoModel.setCompany(entry.get("Company"));
-        contactUsInfoModel.setPhone(entry.get("Phone"));
-        contactUsInfoModel.setComment(entry.get("Comment"));
-        return contactUsInfoModel;
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @DefaultDataTableEntryTransformer
+    public ContactUsInfo getContactUsInfo(Map<String, String> entry, Type toValueType) {
```

Comment:
>Now we do not need to create separate transformations for each model.
how could we use this transformation for a different model, if the return type is `ContactUsInfo` ?

Reply:
my goodness... I'm so stupid.
fixed in 93a0102